src/main/java/by/exadel/internship/service/impl/UserServiceImpl.java

```diff
@@ -41,6 +41,8 @@
 
     public UserDTO getById(UUID id) {
 
+        putClassNameInMDC();
```

Comment:
you can make this method static, add class agrument, move it to utils and reuse in all nessesary classes

Reply:
Fixed